recipes/supertree/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "supertree" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0a802b89b9358b437d0b4cdec69ffb8f8babcc16005ffe017464218a915aaa9e
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
+  run:
+    - python >=3.9
+    - numpy
+    - pandas
+    - IPython
+
+test:
+  imports:
+    - supertree
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/mljar/supertree
+  summary: Interactive Decision Tree Visualization
+  license: LicenseRef-LICENSE.txt
+  license_file: 
+    - LICENSE.txt
+    - supertree-non-commercial-license.pdf 
+    - supertree-commercial-license.pdf
```

Comment:
I'm not a lawyer and I'm always a bit scared o licenses like these. See https://github.com/mljar/supertree/blob/main/supertree-commercial-license.pdf for the full text.

@conda-forge/core any advice here? While conda-forge would not use this in a comercial sense, some users may and, in that case, who is liable? The package provider or just the end user?

Reply:
Hi @ocefpaf and @xhochy,

Thanks for checking the package and licenses. As far as I know conda-forge is a distribution channel. It is not a part in the license. License is provided by MLJAR Sp. z o.o. to the end users. 

What is the main concern to merge the package? Thank you!

Reply:
Distribution must be explicitly included in a license; otherwise, we are not legally allowed to do it.

Reply:
Thank you @xhochy. We will update license and let you know.